client/src/pages/dashboard/Dashboard.js

```diff
@@ -139,18 +152,20 @@ function Dashboard() {
           <div className="proposal-list">
             {proposalList}
           </div>
-          <div class="shadows" aria-hidden="true"></div>
+          <div className="shadows" aria-hidden="true"></div>
         </div>
         <div className="right-proposals">
           <div className="proposal-description">
               <div className="proposal-head-title">{proposalTitle}</div>
               <div className="proposal-head-description">{proposalDescription}</div>
               <div className="dividing-line"> </div>
               <div className="comment-area">
-                <textarea id="textbox" className="comment-box" placeholder="Please enter text here!" rows="7" cols="53"></textarea>
+                <textarea id="textbox" name="textbox" className="comment-box" placeholder="Please enter text here!" rows="7" cols="53"></textarea>
                 <div className="discussion-buttons">
-                  <button className="post-comment">Post Comment</button>
-                  <button className="view-discussion">View Discussion</button>
+                  <button className="post-comment" onClick={postComment}>Post Comment</button>
```

Comment:
As a small quality of life change, it would be cool to have the textbox clear whenever you click "Post" and for there to be some visual notification that your comment was posted!

Having the textbox clear should be pretty simple if you make the change that I mentioned above about using "controlled components" for your `textarea`. Just modify the state variable to be empty!